packages/babel-preset-react-app/index.js

```diff
@@ -115,6 +115,16 @@ module.exports = function(api, opts) {
           regenerator: true,
         },
       ],
+      [
+        require('babel-plugin-named-asset-import'),
```

Comment:
This shouldn't be enabled in the test environment.

Reply:
I was just about to ask for suggestions on how to fix these tests. 😀

I was trying to find a way to use Jest's `moduleNameMapper` to do something with the webpack loader strings but I will try just disabling the plugin in the test environment.